sinks/external_test.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	kube_api "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	fuzz "github.com/google/gofuzz"
+	_ "k8s.io/heapster/api/v1/types"
```

Comment:
I have observed this test being fuzzy even at the current master. Not sure what the cause is